Mobile/Core/IAssetReader.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.IO;
+using Android;
+
+namespace Core
+{
+	public interface IAssetReader
+	{
+		string ReadFileAsString(string fileName);
```

Comment:
Don't you need to have a path argument ? So we can inject it when working on iOS or it used the same path for both OS ?